manifests/profile/tsm.pp

```diff
@@ -41,7 +41,8 @@
   Array[String] $virtualmountpoints = ['/etc','/opt','/var'],
   Array[String] $exclude_dirs = ['/afs/','/net/','/nfs/','/usr/vice/cache/']
 ) {
-  package { 'tivsm-ba': }
+
+  ensure_packages(['tivsm-ba', 'tivsm-api64', 'gskcrypt64', 'gskssl64',])
```

Comment:
We should explore why requiring the single package was not enough to pull in the other three as dependencies. It worked on Debian Stretch.

Reply:
Going to double-check this in case it was an artifact of other transient issues.